src/wallet/rpcdump.cpp

```diff
@@ -383,7 +383,8 @@ UniValue importprunedfunds(const JSONRPCRequest& request)
         throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "Something wrong with merkleblock");
     }
 
-    wtx.SetConf(CWalletTx::Status::CONFIRMED, merkleBlock.header.GetHash(), txnIndex);
+    CWalletTx::Confirmation confirm(CWalletTx::Status::CONFIRMED, merkleBlock.header.GetHash(), txnIndex);
+    wtx.m_confirm = confirm;
```

Comment:
In commit "Replace CWalletTx::SetConf by Confirmation initialization list" (d0c4ce686ba388b9fefff2b153358e33a042ee80)

Not important, but might want to avoid creating named variable with:

```
wtx.m_confirm = CWalletTx::Confirmation{CWalletTx::Status::CONFIRMED, merkleBlock.header.GetHash(), txnIndex};
```

Similarly with all the other temporary `confirm` variables below.